packages/near-sdk-js/src/borsh.ts

```diff
@@ -0,0 +1,9 @@
+import * as borsh from 'borsh';
+import {Schema} from "borsh";
+import {DecodeTypes} from "borsh/lib/types/types";
+export function borshSerialize(schema: Schema, value: unknown, checkSchema?: boolean): Uint8Array {
+    return borsh.serialize(schema, value, checkSchema);
+}
+export function borshDeserialize(schema: Schema, buffer: Uint8Array, checkSchema?: boolean): DecodeTypes {
+    return borsh.deserialize(schema, buffer, checkSchema);
+}
```

Comment:
Let's not make this part of near-sdk-js. Because not all contracts will use borsh. For those who use, they can import it and for those who don't use, they can have a smaller contract to save storage. For this PR, let's either move this code to the borsh example.

Reply:
ok